native-image-tests/src/main/resources/META-INF/native-image/native-image-tests/reflect-config.json

```diff
@@ -0,0 +1,215 @@
+[
+{
+  "name":"apple.security.AppleProvider",
```

Comment:
For example, presumably this is platform-dependent.

Reply:
Yep - to be clear this comes from your gradle task, not from anything I've done.

Reply:
I can see a lot of benefits to start with the run tests twice approach, but long term a few drawbacks.

1) If the Graal JVM is different than native image, then tests may break when they exercise different code paths anyway.
2) It's 2x slower
3) It may hide problems that we want surfaced, e.g. some of these should maybe be included in the config files baked inside of okhttp-sse jar.